app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -134,9 +134,9 @@ def raw_status
     raise MiqException::MiqOrchestrationStatusError, err.to_s, err.backtrace
   end
 
-  def raw_stdout
+  def raw_stdout(format = 'txt')
```

Comment:
Hi @bzwei can I bother you to update this to:

```
def raw_stdout(format = nil)
  format ||= 'txt'
  ...
```

incoming parameter will be nil if not specified in API, this change would allow that to work too without have me to have separate call raw_stdout  vs.  raw_stdout(with_something).

Thanks